.Rbuildignore

```diff
@@ -0,0 +1,2 @@
+^.*\.Rproj$
+^\.Rproj\.user$
```

Comment:
not exactly sure how to use this file just yet but it seems like it's here to ignore certain files when you want to build / compile your R